modules/preview-react/pill/stories/examples/WithList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {Pill} from '@workday/canvas-kit-preview-react/pill';
+import {HStack} from '@workday/canvas-kit-react';
+
+const data = [
+  'Shoes',
+  'Pants',
+  'Dress Shoes',
+  'Color',
+  'Accessories',
+  'Luxury',
+  'Casual',
+  'Hats',
+  'Beanies',
+  'Glasses',
+  'Jewelry',
+];
+
+export const ListOfPills = () => (
+  <HStack spacing="zero" flexWrap="wrap">
```

Comment:
If you're not going to use the `spacing`, then `Flex is probably a better fit here.
```suggestion
  <Flex flexWrap="wrap">
```